### ISSUE TITLE
Group capstone: set up your Kanban board

### ISSUE THREAD

**Issue by rubydevi** [rubydevi/meal-capstone#21]
**Description:**
We are a group of two, and here is the link for our [GitHub project](https://github.com/rubydevi/meal-capstone/projects/1)

**Changes Made:**
- We created an empty repository for our group project.
- We followed the steps described in [curriculum-tools-copy-projects](https://github.com/microverseinc/curriculum-tools-copy-projects).
- Made sure that in the card titles, there are: Student A, Student B, or Student C.
- Student A: @rubydevi 
  Student B: @sadaf-Daneshgar

**Comment by V-Blaze** [comment]
## Hi @rubydevi , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

![almost-there](https://user-images.githubusercontent.com/107632780/229242540-644ade4a-fe15-4f4c-a38b-6ded70cc4e8b.gif)


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Project cards created✔
- Some cards assigned👋 ✔

 ## Required Changes ♻️
- [ ] You have done so great so far But there are some requirements you need to complete before moving to the next milestone. :+1: :100: 
- - There are two students in your group, as required you need to remove all the cards for the `Student C` cards.
- - You have done well assigning the cards to the correct students, you also need to assign all the group cards to both students or one of the students. :smile: :-1:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OyePriscilla** [comment]
Hi @sadaf-Daneshgar and @rubydevi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
![200](https://user-images.githubusercontent.com/103386183/196424328-985f6e98-f7c6-4cf9-bafe-158436f70560.gif)

## Highlights
Assigned the group tasks fairly ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rubydevi** [comment]
Thank you so much for the reviews, I will close the issue now.
